package.json

```diff
@@ -64,15 +66,12 @@
     "ramda": "^0.25.0"
   },
   "devDependencies": {
+    "@fimbul/bifrost": "^0.9.0",
```

Comment:
What's this for?

Reply:
@astroash

Reply:
Oops, that's a tslint dependency, I'll stick it in the other repo